src/niscope/system_tests/test_system_niscope.py

```diff
@@ -60,7 +61,7 @@ def test_fetch_binary8_into(session):
     session.configure_vertical(test_voltage, niscope.VerticalCoupling.AC)
     session.configure_horizontal_timing(50000000, test_record_length, 50.0, 1, True)
     with session.initiate():
-        wfm_infos = session.channels[test_channels].fetch_into(timeout=5.0, wfm=wfm)
+        wfm_infos = session.channels[test_channels].fetch_into(timeout=datetime.timedelta(seconds=5.0), wfm=wfm)
```

Comment:
I feel like we should rely on default timeouts more in the system tests, so that those get exercised.

Reply:
Removed